astropy/modeling/powerlaws.py

```diff
@@ -492,6 +491,14 @@ class Schechter1D(Fittable1DModel):
     power-law form of the function cuts off. Note that the default value for
     ``l_star`` is in units of solar luminosities, in erg/s, to avoid overflow
     errors.
+
+    References
+    ----------
+    
+    .. _ref-schechter1976:
```

Comment:
@bsipocz - I had previously only referenced the paper in the notes section, using the same format as seen in the Voigt1D model. @crawfordsm suggested I add a reference section and I followed the format of blackbody.py. Should I add back the full paper reference in the 'notes' section and also keep the reference section (containing only a hyperlink as you suggest), or what do you think is best? Thanks!